### DISCUSSION TITLE
when i get my student developer pack ?

### DISCUSSION THREAD

**Discussion by kushagra0000** [community/community#68926] [GitHub Education] [answered]
### Select Topic Area

Question

### Body

i sent the request on 29 sep for my student developer pack and it was acceptes and approve on that date but i will not get student developer pack  
![Screenshot_20231001_190228](https://github.com/community/community/assets/146434225/4a5bed31-2fb1-4f4d-991a-748c1e435f82)

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @kushagra0000! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!

**Comment by mantasu** [comment, marked as answer]
Hey, I had the same issue, you just have to wait for a few more days as there will be some time before your benefits become active (I read that it might even take a few weeks). When "Approved" tag becomes purple, benefits should be available:

![Screenshot 2023-09-24 072637](https://github.com/community/community/assets/56790921/d80b71bb-8d34-424d-a682-39a0f986389c)

If you don't receive any benefits after 2 weeks or so, I would suggest opening a [ticket](https://support.github.com/request/education).
